client/state/index.js

```diff
@@ -71,6 +72,7 @@ export const reducer = combineReducers( {
 	componentsUsageStats,
 	countryStates,
 	currentUser,
+	directly,
```

Comment:
What about putting this inside `support/directly` or `help/directly`? It's not immediately clear what it is about.